pkg/sql/parser/sql.y

```diff
@@ -600,7 +603,7 @@ func (u *sqlSymUnion) typeReferences() []tree.ResolvableTypeReference {
 %token <str> NONE NORMAL NOT NOTHING NOTNULL NOWAIT NULL NULLIF NULLS NUMERIC
 
 %token <str> OF OFF OFFSET OID OIDS OIDVECTOR ON ONLY OPT OPTION OPTIONS OR
-%token <str> ORDER ORDINALITY OTHERS OUT OUTER OVER OVERLAPS OVERLAY OWNED OPERATOR
+%token <str> ORDER ORDINALITY OTHERS OUT OUTER OVER OVERLAPS OVERLAY OWNED OWNER OPERATOR
```

Comment:
haha this is definitely already out of order.

Reply:
F